File: Baekjoon/13699.cpp
#include<iostream>
#include<algorithm>
#include<vector>
using namespace std;

using ll = long long;
int main () {
  int n;
  cin >> n;
  vector<vector<ll>> bino(71, vector<ll>(71, 0));
  bino[0][0] = bino[1][0] = bino[1][1] = 1;
  for(int i=2; i<=2*n; i++) {
    for(int j=0; j<=i; j++) {
      bino[i][j] = bino[i-1][j] + (j > 0 ? bino[i-1][j-1] : 0);
    }
  }
  cout << bino[2*n][n] / (n + 1) << endl;
  return 0;
}

File: Baekjoon/13699.py
n = int(input())
bino = [[1]]
for i in range(1, 2 * n + 1):
    bino.append([])
    for j in range(i + 1):
        if j == 0:
            bino[i].append(1)
        elif j == i:
            bino[i].append(bino[i-1][j-1])
        else:
            bino[i].append(bino[i-1][j-1] + bino[i-1][j])
print(bino[2*n][n] // (n + 1))
